Owner: Terje Stoeback <terjes@opera.com>

Adopted from old npapi.h.

--- tmp-npapi.h	2011-09-02 14:59:36.882493949 +0200
+++ npapi.h	2011-09-02 14:59:41.665840423 +0200
@@ -38,6 +38,11 @@
 #ifndef npapi_h_
 #define npapi_h_
 
+// Set MOZ_PLATFORM_MAEMO to a dummy value to avoid BREW compiler warning:
+//
+//   "modules\ns4plugins\src\plug-inc\upstream-npapi.h", line 382: Warning: C2206W: Undefined macro 'MOZ_PLATFORM_MAEMO' in #if - treated as 0
+#define MOZ_PLATFORM_MAEMO -1
+
 #if defined(__OS2__)
 #pragma pack(1)
 #endif
